src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,94 @@
+- The ninja visor can now alternate between normal vision and night vision.
+
+# New Objectives & Mechanics
+
+Gives the Space Ninja more to do and tinker with, so that all rounds don't play out the same.
+
+## Objective Additions
+
+- Steal X spesos - A strongbox for Cargo containing all their deposited spesos will be located in the vault, the Space Ninja is tasked with stealing a designated amount - An alternative to "Steal X technologies".
+- Upload the law board - An AI law board with a Spider Clan-oriented law set, the Space Ninja is tasked with uploading it to the AI upload console - An alternative to "Detonate your spider charge".
+- Hack Si-XXXX - The Space Ninja's kill-target equivalent, they are tasked with hacking a specific cyborg, turning it into a zealous Spider Clan tech-priest that obsesses over hoarding all manners of electronics and machines.
+
+## Objective Changes
+
+- All stealing objectives can now only be performed with a specialized item that is dropped by the ninja upon death, unlike previously where they only used their gloves - This allows the crew to regain their losses.
+- Make an announcement - The communications computer objective now revolves around the Space Ninja sending a Spider Clan Announcement from it, the contents of said announcement being up to their discretion. Like before, it requires the use of turned on space ninja gloves, the one key difference being that now the EMAG interaction is instantaneous, overriding all access restrictions as well as unlocking Spider Clan as an announcement sender. (Replaces "Call in a threat", the new objective keeping the threat-calling functionality is still in discussion)
+- Hack X - 1-3 of the Miscellaneous Interactions machines is picked to be hacked by the Space Ninja. Note that you can still hack machines that aren't included in your objectives. (Replaces "Set Everyone to Wanted")
+
+## Miscellaneous Interactions
+
+- Hacking the criminal records computer is now one of the many miscellaneous actions the Space Ninja can do in order to frustrate the crew - The error announcement will now only be sent by the criminal records computer on the Security channel, opposed to the previous station-wide Central Command Announcement.
+- The crew monitoring server is now prone to being hacked by the Space Ninja, doing so will disable everyone's sensors until deconstructed and reconstructed.
+- The station records computer is now prone to being hacked by the Space Ninja, doing so will delete a small % of random personnel from it accompanied by a station-wide announcement.
+- The camera router is now prone to being hacked by the Space Ninja, doing so will kill a large % of its connected cameras and render them useless.
```

Comment:
I'd like the server to become unrepairable, and require an engineer or so to do maintenance on the camera router. It can be something where there isn't no true physical destruction, it's just disrupted and required repair.

This would make countering the AI _extremely viable_ and the ninja could plan ahead: "I'll break into telecoms and short out the cameras to security, so I can get a high-cap without being spotted"

To reduce metagaming ("ninja on station, my cameras in sec aren't working!!") we could have this same effect trigger for a random telecoms server during a solar flare. It would expose players to the situation more often and make it less unique to the ninja. EMPs should also disrupt cameras, and if they hit servers, it should disrupt them and cut cams too. I believe there's an issue on the repo right now that asks the question "should cameras have to be routed through a router" which I really like, both for regular gameplay, and for this idea.

Reply:
> I'd like the server to become unrepairable, and require an engineer or so to do maintenance on the camera router. It can be something where there isn't no true physical destruction, it's just disrupted and required repair.

Maybe, just maybe, what if instead of the router itself being affected only the cameras require repairing? It'd probably require for a certain % of them to suddenly get their wires cut, though. I just can't really see any way of the engineers having a way to repair routers without it just being a "use in-case of ninja" thing.

> This would make countering the AI _extremely viable_ and the ninja could plan ahead: "I'll break into telecoms and short out the cameras to security, so I can get a high-cap without being spotted"
> 
> To reduce metagaming ("ninja on station, my cameras in sec aren't working!!") we could have this same effect trigger for a random telecoms server during a solar flare. It would expose players to the situation more often and make it less unique to the ninja. EMPs should also disrupt cameras, and if they hit servers, it should disrupt them and cut cams too. I believe there's an issue on the repo right now that asks the question "should cameras have to be routed through a router" which I really like, both for regular gameplay, and for this idea.

agreed! it'd also enrich engineering gameplay which they severely need, also, if possible, could you link said question so I can take a look at the discussions that take place?

Reply:
cameras dont even need to be set up for AI to see through them, let alone need a working router

Reply:
> cameras dont even need to be set up for AI to see through them, let alone need a working router

yeah that's pretty weird, though Isn't this mostly due to AI atm being extremely barebones? Plus, this is also troublesome for people like the Warden, who are also constantly checking cameras.